packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -0,0 +1,48 @@
+const fs = require(`fs`)
+const { onPostBootstrap } = require(`../gatsby-node`)
+
+jest.mock(`fs`)
+
+
+describe(`Test plugin manifest options`, () => {
+  it(`correctly works with default parameters`, (done) => {
+    fs.existsSync.mockReturnValue(true)
+    fs.writeFileSync = jest.fn()
+    onPostBootstrap([], {
```

Comment:
You can use `async/await` here instead of done, which we tend to do more often!

Reply:
test is really mostly the same, just ends up looking like this

```js
it(`correctly works with default parameters`, async () => {
  // other stuff
  await onPostBootstrap()

  expect(fs.writeFileSync.mock.calls).toMatchSnapshot() // I'd grab the actual value we're looking for here, instead of the whole mock
})
```

Reply:
yes done!